src/commands/get/release-line.js

```diff
@@ -0,0 +1,79 @@
+const chalk = require('chalk');
+const Command = require('../../base');
+
+class ReleaseLineCommand extends Command {
+
+  /**
+   * Renders release line information for a package.
+   *
+   * @param {Object} releaseLine - Release line information
+   */
+  render(releaseLine) {
+    this.log('');
+    this.log(chalk.green.bold(`${releaseLine.pkg} | ${releaseLine.version}`));
+    this.log('');
+
+    if (releaseLine.previousVersion) {
+      this.log('Previous version: ', releaseLine.previousVersion);
+      this.log('');
+    }
+
+    if(releaseLine.dependents){
+      this.log('Dependents: ');
+      for(let dependant in releaseLine.dependents) {
+        this.log(`  - ${dependant}@${releaseLine.dependents[dependant]}`);
+      }
+      this.log('');
+    }
+  }
+
+  /**
+   * Runs the get:release-line command
+   *
+   * @async
+   * @returns {Promise} The result or error
+   */
+  async run() {
+    const { flags, args } = this.parse(ReleaseLineCommand);
+    const { package: packageArg } = args;
+    const { pkg, version } = this.parsePackage(packageArg);
+    const { wrhsHost, auth } = this.mergeConfig(flags);
+
+    if (!wrhsHost) {
+      return this.error(this.missingHostError());
+    }
+
+    const wrhs = this.wrhs(auth, { wrhsHost });
+
+    return new Promise((resolve, reject) => {
+      wrhs.releaseLine.get({ pkg, version }, (err, releaseLine) => {
```

Comment:
I assume this is okay with an undefined version?

Reply:
Yea the client handles it